examples/agent_motion_prediction/agent_motion_prediction.ipynb

```diff
@@ -47,7 +47,7 @@
    "outputs": [],
    "source": [
     "# set env variable for data\n",
-    "os.environ[\"L5KIT_DATA_FOLDER\"] = \"PATH_TO_YOUR_DATA\"\n",
+    "os.environ[\"L5KIT_DATA_FOLDER\"] = \"/tmp/l5kit_data\"\n",
```

Comment:
I would not recommend user to save data in `/tmp` folder, which means you need to re-download data after reboot. I would either let user to decide `PATH_TO_YOUR_DATA ` or some permanent storing place

Reply:
I know, I just forgot to reset it :)